translator/defaultKeyCase.go

```diff
@@ -57,6 +57,19 @@ func DefaultIntegralCase(key string, defaultVal, input interface{}) (returnKey s
 	return
 }
 
+func DefaultInteger64Case(key string, defaultVal, input interface{}) (returnKey string, returnVal interface{}) {
```

Comment:
I am uncertain if this is a little cargo cult programming. We do need to parse from float64, but I wonder if we need this wrapper function or should we move this logic to the calling function.